sql/types/json_encode.go

```diff
@@ -314,6 +316,7 @@ func writeMarshalledValue(writer io.Writer, val interface{}) error {
 		writer.Write([]byte{'"'})
 		return nil
 	case json.Marshaler:
+		decimal.MarshalJSONWithoutQuotes = true
```

Comment:
would we want to do this on init? seems like we'd want the lifecycle to be predictable

Reply:
We probably do want to do it on init, but I wasn't sure where the best place to do that would be. This is the only place where we marshal decimals (I think...) so it should be predictable, but confusing and easy to make unpredictable in the future.

Do you have a suggestion for where I should do this?

Reply:
my first thought was top of file in the startup routine `func init() {}`?

Reply:
Ah. Not sure how I missed that. Fixed.